### PR TITLE
Summation function, sum.py

### PR DESCRIPTION
**Pull request by Adexxy**
added summation function, sum.py

### CODE REVIEW

File: kaisen/kaisen/src/taiwo/sum.py
Comment (tofetpuzo) [inline kaisen/kaisen/src/taiwo/sum.py:1]: please can you make parameters num1 and num2 parameters to the function?

Comment (Adexxy) [reply kaisen/kaisen/src/taiwo/sum.py:1]: Do you mean include them inside the function like this?:
def suml():
    num1 = 2000
    num2 = 3400
    total = num1 + num2
    return total
print(suml())

File: kaisen/kaisen/src/taiwo/sum.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
-num1 = 2000
-num2 = 3400
 
 def suml():
```
Comment (tofetpuzo) [inline kaisen/kaisen/src/taiwo/sum.py:2]: for example
```

def sum1(num1, num2):
      .......
```

Comment (Adexxy) [reply kaisen/kaisen/src/taiwo/sum.py:2]: okay, got it.

Comment (Adexxy) [reply kaisen/kaisen/src/taiwo/sum.py]: Done

**Review by tofetpuzo — APPROVED** [review]

**Review by tofetpuzo — APPROVED** [review]

**Review by tofetpuzo — APPROVED** [review]
looks good

**Pull request merged**